scripts/browser.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+[[ -z "$NGIO_ENV_DEFS" ]] && . ./scripts/env-set.sh > /dev/null
+
+if [[ $TASK == *test* ]]; then
+  set -x
+  sh -e /etc/init.d/xvfb start
+  #t=0; until (xdpyinfo -display :99 &> /dev/null || test $t -gt 10); do sleep 1; let t=$t+1; done
+  # content_shell --run-layout-test --disable-gpu-early-init --disable-gpu
```

Comment:
This script doesn't actually launch a browser or anything else, does it? Just xvfb.

Reply:
Right

Reply:
I'd suggest renaming it to `xvfb.sh` or something similar, then. Unless you have a concrete plan of uncommenting that `content_shell` line.

Reply:
Ok